code/test/CoreTemplateStudio.Core.Test/Helpers/LogFixture.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace Microsoft.Templates.Core.Test.Helpers.FsTests.Helpers
+{
+    [SuppressMessage("Design", "CA1063:Implement IDisposable Correctly", Justification = "Testing purposes only")]
+    public class LogFixture : IDisposable
```

Comment:
This should probably be called FSTestsFixture as it sets the folderpath and handles log files.